builder/oracle/oci/config.go

```diff
@@ -61,7 +61,9 @@ type Config struct {
 	ImageName   string `mapstructure:"image_name"`
 
 	// Instance
-	InstanceName string `mapstructure:"instance_name"`
+	InstanceName        string                            `mapstructure:"instance_name"`
+	InstanceTags        map[string]string                 `mapstructure:"instance_tags"`
+	InstanceDefinedTags map[string]map[string]interface{} `mapstructure:"instance_defined_tags"`
```

Comment:
Nice PR, thanks for opening !

In order to also be able to add singular tags in hcl like:
```hcl
instance_tag {
  key = "my_key"
  value = "my_value"
}
```
Which will allow setting tags from variables using the `dynamic` stanza.

I would also add:

```go
	InstanceTag        hcl2template.KeyValues `mapstructure:"instance_tags" required:"false"`
	InstanceDefinedTag hcl2template.KeyValues `mapstructure:"instance_tags" required:"false"`
```

If you do so don't forget to call `InstanceTag.CopyOn(InstanceTags)...)` so they can be seen.

Reply:
This is a cool feature. However, I'd like to keep this PR simple, and as nothing else in the OCI builder uses this feature, I'll leave adding it in to another PR, if that's alright.

Reply:
Got it !